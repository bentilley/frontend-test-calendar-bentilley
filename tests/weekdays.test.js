import React from 'react';
import renderer from 'react-test-renderer';

import Weekdays from 'components/calendar/weekdays';

let component;

describe('Weekdays component', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Weekdays />);
    const instance = component.getInstance();
    const result = instance.render();
    expect(result).toMatchSnapshot();
  });
});
