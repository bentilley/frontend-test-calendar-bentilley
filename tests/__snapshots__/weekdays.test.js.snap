// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weekdays component renders correctly 1`] = `
<div
  className="weekdays"
>
  <div
    className="cell"
  >
    MON
  </div>
  <div
    className="cell"
  >
    TUE
  </div>
  <div
    className="cell"
  >
    WED
  </div>
  <div
    className="cell"
  >
    THU
  </div>
  <div
    className="cell"
  >
    FRI
  </div>
  <div
    className="cell"
  >
    SAT
  </div>
  <div
    className="cell"
  >
    SUN
  </div>
</div>
`;
